pkg/ebpf/tracee.go

```diff
@@ -119,6 +119,8 @@ type Tracee struct {
 	readyCallback   func(gocontext.Context)
 	// Streams
 	streamsManager *streams.StreamsManager
+	// policyManager manages rules enabled/disable per policy
```

Comment:
In the near future, it will also mange policy state, including state, so maybe we better not restrict the comment to rule enable/disable